packages/market-matcher/package.json

```diff
@@ -14,20 +14,15 @@
     "version": "1.1.0",
     "main": "dist/js/src/index.js",
     "scripts": {
-        "build": "yarn generate-schemas && yarn build-ts",
+        "build": "yarn build-ts",
```

Comment:
Can we just replace `build-ts` with `build`?

Reply:
So build should clean and build-ts actually, which is here omitted.